_docs/compliance/meeting-tic-requirements.md

```diff
@@ -49,7 +64,7 @@ APIRouter -->|Change app| App
 
 Any traffic from an agency-authorized boundary (eg physical network of an agency building, or collective virtual boundary for all networked agency buildings) to one that it is not under your agency’s control (e.g. the open internet, or cloud.gov) is likely already routed over a TIC connection.
 
-For requests originating from your agency’s TIC egress bound for cloud.gov, our TLS implementation plays the role of the orange "encrypted tunnel" in Figure 14 of the [TIC reference architecture](https://s3.amazonaws.com/sitesusa/wp-content/uploads/sites/482/2015/04/TIC_Ref_Arch_v2-0_2013.pdf). You will access both api.fr.cloud.gov and yourapp.app.cloud.gov exclusively over TLS because the cloud.gov domain is included in [the HSTS preload list](https://hstspreload.org/) for your browser. (If you try to directly access those domains via HTTP, your request will be 301 redirected over to HTTPS; it's not possible to get any other response without TLS.)
+For requests originating from your agency’s TIC egress bound for cloud.gov, our TLS implementation plays the role of the orange "encrypted tunnel" in Figure 14 of the [TIC 2.0 reference architecture](https://s3.amazonaws.com/sitesusa/wp-content/uploads/sites/482/2015/04/TIC_Ref_Arch_v2-0_2013.pdf). You will access both api.fr.cloud.gov and yourapp.app.cloud.gov exclusively over TLS because the cloud.gov domain is included in [the HSTS preload list](https://hstspreload.org/) for your browser. (If you try to directly access those domains via HTTP, your request will be 301 redirected over to HTTPS; it's not possible to get any other response without TLS.)
```

Comment:
When I try and access the PDF document linked in this section I get an error. Downloading by other means also results in an error. Not sure if it's me or the link (or doc) is broken.